docs/start_guide.md

```diff
@@ -0,0 +1,279 @@
+
+If you are on a [Project Atomic host](https://projectatomic.io/download) you can interact with `atomicapp` via the `atomic` cli command.
+
+Some commands for `atomicapp` on an atomic host are a bit different.
+
+However. Regardsless of the `atomic run` command, a `--mode` can be passed to change the functionality of the command. 
+
+`atomicapp fetch projectatomic/helloapache`
+
+vs
+
+`atomic run projectatomic/helloapache --mode fetch`
+
+`atomicapp run projectatomic/helloapache`
+
+vs
+
+`atomic run projectatomic/helloapache`
+
+`atomicapp stop ./myappdir`
+
+vs
+
+`atomic stop projectatomic/helloapache ./myappdir`
+
+`atomicapp genanswers ./myappdir`
+
+vs
+
+`atomic run projectatomic/helloapache ./myappdir --mode genanswers`
+
+## Building your first Atomic App
+
+A typical Atomic App or "Nulecule" container consists of the following files:
+```sh
+~/helloapache
+▶ tree
+.
+├── answers.conf.sample
+├── artifacts
+│   ├── docker
+│   │   └── hello-apache-pod_run
+│   ├── kubernetes
+│   │   └── hello-apache-pod.json
+│   └── marathon
+│       └── helloapache.json
+├── Dockerfile
+├── Nulecule
+└── README.md
+```
+
+We will go through each file and folder as we build our first Atomic App container.
+
+For this example, we will be using the [helloapache](https://github.com/projectatomic/nulecule-library/tree/master/helloapache) example from the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
+
+In order to follow along, fetch the container and `cd` into the directory:
+```sh
+atomicapp fetch --destination helloapache projectatomic/helloapache
+cd helloapache
+```
+
+### ~/helloapache/Dockerfile
+Atomic App itself is packaged as a container. End-users typically do not install the software from source, instead using the `atomicapp` container as the `FROM` line in a Dockerfile and packaging your application on top. For example:
+
+
+```Dockerfile
+FROM projectatomic/atomicapp
+
+MAINTAINER Your Name <you@example.com>
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+Within `helloapache` we specify a bit more within our labels:
+```Dockerfile
+FROM projectatomic/atomicapp:0.4.2
+
+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>
+
+LABEL io.projectatomic.nulecule.providers="kubernetes,docker,marathon" \
+      io.projectatomic.nulecule.specversion="0.0.2"
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+__Optionally__, you may indicate what providers you specifically support via the Docker LABEL command.
+
+__NOTE:__ The Dockerfile you supply here is for building the atomicapp image, and is not the Dockerfile you use to build your upstream application image. Per the examples, your upstream application should be already available as a container image or as several different images, in the Atomic Registry, the Docker Hub, or whatever image registry is appropriate.
```

Comment:
is for building the Nuleculized application container image (sometimes called an Atomic App). It is not the Dockerfile you use to build the Atomic App software container image. The Atomic App software container image should already be built at this time and is what you are specifying in the `FROM projectatomic/atomicapp` line. 

etc.. etc.. 
I think you can incorporate some of what I wrote to make it clearer.